src/zeit/importer/article.py

```diff
@@ -57,6 +60,15 @@ def _transform(xslt, xml, **kwargs):
     return xslt(xml, **kwargs)
 
 
+def map_access(context, text):
```

Comment:
Ach nee. Das ergibt keinen Sinn... Das "nicht für online" soll ja bei der ersten Stufe noch da sein... (Wo ist die nächste Tischkante) ... 
Also: Ich könnte in der ersten Stufe die bekannten Werte mappen und die unbekannten lasse ich wie sie sind. In der zweiten Stufe skippe ich dann den Import für Artikel mit unbekannten Access Werten.